### PR TITLE
Fix the update-collections script - bump release in spec and build vendored_collections.yml correctly

### LINKED ISSUE #140: update_vendored_collection does not update Release field in spec file
https://src.fedoraproject.org/rpms/linux-system-roles/c/a38c7ee816ae481403fe9d5aa84bb3515d0591a1?branch=rawhide

The changelog has `1.16.0-2` but the `Release:` field in the spec file is still `Release: 1%{?dist}`

### PR DESCRIPTION
**Pull request by spetrosi**
Fixes #140

### CODE REVIEW

File: update_vendored_collections/update_vendored_collections.py [hunk 1]
```diff
@@ -169,15 +180,22 @@ def repo_force_push(repo, remote, branch):
...
         collections_versions.update({collection: version})
     for coll in hsh["collections"]:
-        for collection, version in collections_versions.items():
-            if coll["name"] == collection:
-                updated_requirements["collections"].append(
-                    {"name": coll["name"], "version": version}
-                )
+        if coll["name"] not in collections_versions.keys():
+            collections_versions.update({coll["name"]: coll["version"]})
+    # Sort collections eto be in the same order as hsh["collections"]
```
Comment (richm) [inline update_vendored_collections/update_vendored_collections.py:190]: "eto" -> "to"

File: update_vendored_collections/update_vendored_collections.py [hunk 2]
```diff
@@ -104,13 +104,24 @@ def spec_add_changelog(content, collection_tarballs, lines):
...
 
 
+def spec_bump_release(content):
+    print("Bumping release in the spec file")
+    cur_release = re.search(r"^Release: (\d+)", content, flags=re.MULTILINE).group(1)
+    new_release = str(int(cur_release) + 1)
+    content = re.sub(
+        r"^Release: \d+", "Release: " + new_release, content, flags=re.MULTILINE
+    )
+    return content
```
Comment (richm) [inline update_vendored_collections/update_vendored_collections.py:114]: You also need to update the release number in the changelog entry

Comment (spetrosi) [reply update_vendored_collections/update_vendored_collections.py:114]: Script does this in spec_add_changelog: https://github.com/linux-system-roles/auto-maintenance/blob/master/update_vendored_collections/update_vendored_collections.py#L95

File: update_vendored_collections/update_vendored_collections.py (hunk 1 above)
Comment (spetrosi) [reply update_vendored_collections/update_vendored_collections.py:190]: Fixed.

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**